client/layout/guided-tours/style.scss

```diff
@@ -1,6 +1,7 @@
 .guided-tours__step {
 	position: fixed;
 	max-width: 410px;
+	width: 410px;
```

Comment:
Could this come back to bite us later? Any other kind of step, or viewport, where we wouldn't want it? If we keep it, can we get rid of `max-width` right above?